step-plans/step-plans-core/pom.xml

```diff
@@ -47,19 +47,17 @@
 		<dependency>
 			<groupId>ch.exense.step</groupId>
 			<artifactId>step-api-json-schema</artifactId>
-			<version>${step-api.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
-			<version>${commons-lang.version}</version>
 		</dependency>
 
 		<!-- Required in compile scope as some classes of main code are using junit -->
+		<!-- FIXME: IS THIS A GOOD IDEA ???? -->
```

Comment:
The "main" class PlanRunnerResultAssert is using junit.framework.Assert. This class is only used then in test classes, I'm not sure if it can be use in productive code (like Step junit runner....). To be checked with @jeromecomte